docs/guide/shortcuts/README.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Mouse and Keyboard Shortcuts
+---
+
+# Mouse and Keyboard Shortcuts
+
+For shortcuts the following convention is used:
+
+* <span class="ui label">C</span> — platform-neutral keyboard input;
+* <span class="ui blue label">Ctrl+C</span> — PC-only keyboard input;
+* <span class="ui grey label">⌘ C</span> — Mac-only keyboard input;
+* <span class="ui olive label">Click</span> — mouse input.
+
+## Project management
+
+<table class="ui celled table">
+  <thead>
+    <tr>
+      <th>Shortcut</th>
+      <th>Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <span class="ui blue label">Ctrl+N</span> |
+        <span class="ui grey label">⌘ N</span>
+      </td>
+      <td>
+        New patch
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="ui blue label">Ctrl+Shift+N</span> |
+        <span class="ui grey label">⇧⌘ N</span>
+      </td>
+      <td>
+        New project
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="ui blue label">Ctrl+S</span> |
+        <span class="ui grey label">⌘ S</span>
+      </td>
+      <td>
+        Save project
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <span class="ui blue label">Ctrl+O</span> |
+        <span class="ui grey label">⌘ O</span>
+      </td>
+      <td>
+        Open project
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Patch editing
```

Comment:
We can also add "Double click" on the Node — opens Node as a patch.

Reply:
Yep. But in a version not yet released. We have to remember to add it right before 0.15.0 release.